src/stream/src/executor/monitor/streaming_stats.rs

```diff
@@ -1044,26 +1049,29 @@ impl StreamingMetrics {
         let materialize_cache_hit_count = register_guarded_int_counter_vec_with_registry!(
             "stream_materialize_cache_hit_count",
             "Materialize executor cache hit count",
-            &["table_id", "actor_id", "fragment_id"],
+            &["actor_id", "table_id", "fragment_id"],
             registry
         )
-        .unwrap();
+        .unwrap()
+        .relabel_debug_1(level);
 
         let materialize_cache_total_count = register_guarded_int_counter_vec_with_registry!(
             "stream_materialize_cache_total_count",
             "Materialize executor cache total operation",
-            &["table_id", "actor_id", "fragment_id"],
+            &["actor_id", "table_id", "fragment_id"],
             registry
         )
-        .unwrap();
+        .unwrap()
+        .relabel_debug_1(level);
 
         let stream_memory_usage = register_guarded_int_gauge_vec_with_registry!(
             "stream_memory_usage",
             "Memory usage for stream executors",
-            &["table_id", "actor_id", "desc"],
+            &["actor_id", "table_id", "desc"],
```

Comment:
Note the changes on the label ordering. As we always suppress a prefix of this slice, the front one should have the largest cardinality.